src/server/page.ts

```diff
@@ -200,11 +199,7 @@ export class Page extends EventEmitter {
   }
 
   async _runAbortableTask<T>(task: (progress: Progress) => Promise<T>, timeout: number): Promise<T> {
-    return runAbortableTask(async progress => {
-      for (const agent of instrumentingAgents)
-        await agent.onBeforePageAction(this, progress);
-      return task(progress);
-    }, timeout);
+    return runAbortableTask(task, timeout);
```

Comment:
Just in line this call?

Reply:
Yeah, I'll follow up with that, to not make this patch even bigger.